src/web/trykkomania/trykkomania.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: Trykkomania
-level: 2
```

Comment:
Ikke enig med denne. Konseptene er vanskelige, men man trenger kun kopiere koden til riktig plass for å få det til. Se beskrivelse av nivå i README.md i rota av repoet.